content/tutorials/coordination.md

```diff
@@ -0,0 +1,132 @@
+``` 
+cd ~/catkin_ws/src/ && git clone https://github.com/prl-mushr/mushr_coordination.git
+cd mushr_coordination && git clone https://github.com/whoenig/libMultiRobotPlanning.git
+```
+
+## API
+For adjusting params see `launch/mushr_coordination.launch` it has the environment params for the planner. Change number of waypoints for each goal by setting parameter `num_waypoint`. Change the car team setting by editing or creating new config file. 
+
+### [Example](https://github.com/prl-mushr/mushr_coordination/blob/main/config/4cars1.yaml)
+```
+num_agent: 4
+car1:
+    name: "car1"
+    color: "FF0000" #red
+car2:
+    name: "car2"
+    color: "00FF00" #green
+car3:
+    name: "car3"
+    color: "0000FF" #blue
+car4:
+    name: "car4"
+    color: "DB921D" #orange
+```
+
+### Publishers
+Topic | Type | Description
+------|------|------------
+`/{car_1's name}/waypoints` | [geometry_msgs/PoseArray](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseArray.htmll)| Pathway assigned to car_1
+...
+`/{car_n's name}/waypoints`| [geometry_msgs/PoseArray](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseArray.htmll) | Pathway assigned to car_n
+`/{car_1's name}/marker` | [visualization_msgs/Marker](http://docs.ros.org/en/api/visualization_msgs/html/msg/Marker.html)| Intermediate points of task(s) assigned to car_1 
+...
+`/{car_n's name}/marker`| [visualization_msgs/Marker](http://docs.ros.org/en/api/visualization_msgs/html/msg/Marker.html) | Intermediate points of task(s) assigned to car_n 
+`/mushr_coodination/border` | [visualization_msgs/Marker](http://docs.ros.org/en/api/visualization_msgs/html/msg/Marker.html) | Borderline of the current setup environment for the planner
+
+
+### Subscribers
+Topic | Type | Description
+------|------|------------
+`/{car_1's name}/init_pose` | [geometry_msgs/PoseStamp](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseStamped.html)| Initial position of car_1
+...
+`/{car_n's name}/init_pose` | [geometry_msgs/PoseStamp](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseStamped.html)| Initial position of car_n
+`/mushr_coordination/obstacles` | [geometry_msgs/PoseArray](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseArray.htmll)| List of obstacles in the map
+`/mushr_coordination/goals` | [/mushr_coordination/GoalPoseArray](#mushr_coordination/GoalPoseArray ) | List of goals for {n} cars to complete
+
+### Message Definition
+#### mushr_coordination/GoalPoseArray  
+[`std_msgs/Header`](http://docs.ros.org/en/melodic/api/std_msgs/html/msg/Header.html) Header \
+`float64` scale `#scala for converting continous space to grid space`   
+`float64` minx \
+`float64` miny \
+`float64` maxx \
+`float64` maxy \
+[`geometry_msgs/PoseArray`](http://docs.ros.org/en/melodic/api/geometry_msgs/html/msg/PoseArray.html)[] goals 
+
+## Running the coordination planner
+
+```
+roslaunch mushr_coordination mushr_coordination.launch cars_file:={car's team config file}
```

Comment:
What does this do? Does it launch rviz? Does it publish paths? Do I need the sim running?

Reply:
pictures